C-Plus-Plus/cp/Paint_Fence.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+___Paint Fence Algorithm___
+
+Problem:
+Given a fence with n posts and k colors
+Find out the number of ways of painting the fence
+such that at most 2 adjacent posts have the same color.
+
+Input:
+n : number of posts
+k : number of colors
+
+Output:
+result: number of ways of painting
+        considering the condition above
+
+*/
+
+#include <iostream>
+using namespace std;
+
+int paint_fence(int n, int k){          //function uses permutation logic to output number of possible ways of painting
```

Comment:
Shift the comment above the function and after shifting just add a new line after `using namespace std;`